server/src/models.js

```diff
@@ -20,12 +20,12 @@ export const defineModels = (db) => {
     version: { type: Sequelize.INTEGER }, // version the password
   });
 
-  const CapabilityModel = db.define('capability', {
-    name: { type: Sequelize.STRING },
-  });
-
   const TagModel = db.define('tag', {
     name: { type: Sequelize.STRING },
+    type: {
+      type: Sequelize.ENUM,
+      values: ['group', 'capability'],
```

Comment:
Are we using "group" to mean "unit/region" in this context? It might be a bit confusing if we also call the type of the tag "group"

Reply:
can you check whether this is actually making an enum type in the DB? If it is I caution against it. Having to do a schema change every time you want to add an extra option to an enum is a pita. A set of constant strings or integer ID that is looked up in a `js` file might be preferable. Remember to optimise for less effort rather than increased server efficiency.